### DISCUSSION TITLE
Feedback for the "Add filter"

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1583] [Anything Else]
Couple of pieces of feedback for the "Add filter":

- Now that we have cities and regions, we should change the name of the filter in the "Add filter" dropdown from "Country" to "Location".

- After speaking to a couple of people that were looking for a search button, I was thinking we could change "Add filter" to just be a search icon in the top row of the dashboard. Perhaps then it would be more obvious that it allows you to search for more people new to Plausible?

**Comment by ukutaht** [comment]
number 1 will go live with the next deploy.

For the icon, what do you think about this? https://github.com/plausible/analytics/pull/1635

**Reply by metmarkosaric** [reply]
Nice, looks good to me!
